packages/components/src/snackbar/test/list.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { render, screen } from '@testing-library/react';
+import { click } from '@ariakit/test';
+
+/**
+ * Internal dependencies
+ */
+import SnackbarList from '../list';
+
+window.scrollTo = jest.fn();
```

Comment:
When the `Snackbar` component is removed, the animation runs through `motion.div`. If we don't mock `window.scrollTo`, the following error seems to occur.

```
expect(jest.fn()).not.toHaveErrored(expected)

Expected mock function not to be called but it was called with:
[[Error: Not implemented: window.scrollTo]],[[Error: Not implemented: window.scrollTo]]
```